android/src/main/kotlin/com/hypertrack/sdk/flutter/common/HyperTrackError.kt

```diff
@@ -0,0 +1,14 @@
+package com.hypertrack.sdk.flutter.common
+
+// enum naming convention is ignored to make datatype sync
```

Comment:
WDYM?

Reply:
I think I don't understand this comment. What is meant by it?

Reply:
In case we change smth in this values we can just copypaste it everywhere without any mapping like LOCATION_SERVICES_DISABLES = "locationServicesDisabled"